docker/vision/Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG BASE_IMAGE
+
+FROM ${BASE_IMAGE}
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+##############################
+# MosaicML Vision Dependencies
+##############################
+
+RUN sudo apt-get update && \
+    sudo apt-get install -y --no-install-recommends \
+    # For FFCV:
+    pkg-config \
+    libturbojpeg-dev \
+    libopencv-dev \
+    # For deeplabv3:
+    ffmpeg \
+    libsm6 \
+    libxext6 && \
+    sudo apt-get autoclean && \
+    sudo apt-get clean && \
+    sudo rm -rf /var/lib/apt/lists/*
+
+RUN sudo pip3 install ffcv opencv-python numba 
+
+RUN sudo pip3 install mmcv-full==1.4.4
+
+RUN sudo pip3 install mmsegmentation
+
```

Comment:
nit: extra newlines

Reply:
@ravi-mosaicml : mosaicml/pytorch_vision has already been created.

Reply:
@ravi-mosaicml : added vision image as stage. Ideally I would have liked to install vision stuff before setting the username in Pytorch image but I don't know a way to share code between stages without duplicating some of the code.